subprojects/docs/src/docs/userguide/reference/command_line_interface.adoc

```diff
@@ -66,17 +66,26 @@ This will run the single "myTask" and also all of its <<tutorial_using_tasks.ado
 
 [[executing_tasks_in_multi_project_builds]]
 === Executing tasks in multi-project builds
-In a <<intro_multi_project_builds.adoc#intro_multi_project_builds, multi-project build>>, subproject tasks can be executed with ":" separating subproject name and task name. The following are equivalent _when run from the root project_:
+
+In a <<intro_multi_project_builds.adoc#intro_multi_project_builds, multi-project build>>, subproject tasks can be executed with ":" separating subproject name and task name.
+The following are equivalent _when run from the root project_:
 ----
 $ gradle :my-subproject:taskName
 $ gradle my-subproject:taskName
 ----
 
-You can also run a task for _all_ subprojects by again using a task _selector_ that consists of the task name only. For example, this will run the "test" task for all subprojects when invoked from the root project directory:
+You can also run a task for _all_ subprojects by again using a task _selector_ that consists of the task name only.
+For example, this will run the "test" task for all subprojects when invoked from the root project directory:
 ----
 $ gradle test
 ----
 
+[NOTE]
+====
+Some tasks selectors, like `help`, will only run the task on the project they are invoked on and not on all the subprojects.
```

Comment:
Maybe we should list _all_ core tasks that have special behavior.

> Some tasks selectors - like `help`, `dependencies`, and `dependencyInsight` - will only run the task on the project they are invoked on and not on all the subprojects.

Reply:
The whole list is longer than that ... it even has tasks I have no idea what they are, like the `ModelReport` ...
So I went with the short version.

Reply:
👍